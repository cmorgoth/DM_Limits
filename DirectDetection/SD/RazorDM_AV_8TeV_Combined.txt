weight 1.0
1	927.949
10	933.017
100	902.102
400	698.033
700	478.246
1000	323.449
