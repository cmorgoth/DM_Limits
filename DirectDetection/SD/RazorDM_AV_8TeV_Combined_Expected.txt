weight 1.0
1	1115.51
10	1109.96
100	1084.98
400	831.812
700	585.828
1000	381.736
